diff -Nru cinelerra.orig//image/cinelerra.desktop cinelerra/image/cinelerra.desktop
--- cinelerra.orig//image/cinelerra.desktop	2011-05-06 15:37:51.000000000 -0400
+++ cinelerra/image/cinelerra.desktop	2011-05-06 15:38:52.197036001 -0400
@@ -7,7 +7,7 @@
 Comment=Video Editor
 Categories=Application;AudioVideo;Video;
 Encoding=UTF-8
-Exec=cinelerra
+Exec=Cinelerra
 Icon=cinelerra.xpm
 Terminal=false
 Type=Application
