diff -Nru cinelerra.orig//image/cinelerra.desktop cinelerra/image/cinelerra.desktop
--- cinelerra.orig//image/cinelerra.desktop	2010-11-24 15:18:04.000000000 -0800
+++ cinelerra/image/cinelerra.desktop	2010-12-08 14:25:28.086131906 -0800
@@ -5,7 +5,7 @@
 Name[fr]=Cinelerra
 Name[es]=Cinelerra
 Comment=Video Editor
-Categories=Application;AudioVideo;Multimedia;
+Categories=Application;AudioVideo;Video;
 Encoding=UTF-8
 Exec=cinelerra
 Icon=cinelerra.xpm
